tools/experiments/experiments.js

```diff
@@ -448,9 +450,41 @@ function showConfirmation_(message, callback) {
   container.classList.add('show');
 }
 
+/**
+ * Loads the AMP_CONFIG objects from whatever the v0.js is that the
+ * user has (depends on whether they opted into canary), so that
+ * experiment state can reflect the default activated experiments.
+ */
+function getAmpConfig() {
+  return new Promise((resolve, reject) => {
+    const xhr = new XMLHttpRequest();
+    xhr.addEventListener('load', () => {
+      resolve(xhr.responseText);
+    });
+    xhr.addEventListener('error', () => {
+      reject(new Error(xhr.statusText));
+    });
+    // Cache bust, so we immediately reflect AMP_CANARY cookie changes.
+    xhr.open('GET', '/v0.js?' + Math.random(), true);
+    xhr.send(null);
+  }).then(text => {
+    const match = text.match(/self\.AMP_CONFIG=([^;]+)/);
+    if (!match) {
+      throw new Error('Can\'t find AMP_CONFIG in: ' + text);
+    }
+    // Setting global var to make standard experiment code just work.
+    return self.AMP_CONFIG = JSON.parse(match[1]);
```

Comment:
I'm reconsidering. I thought we'd actually have the ability to turn them off, but that doesn't appear to be the case right now. So, adding `disabled` may make sense.